src/main/java/com/google/api/codegen/transformer/nodejs/NodeJSParamDocTransformer.java

```diff
@@ -0,0 +1,172 @@
+/* Copyright 2016 Google Inc
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.google.api.codegen.transformer.nodejs;
+
+import com.google.api.codegen.config.MethodConfig;
+import com.google.api.codegen.transformer.MethodTransformerContext;
+import com.google.api.codegen.transformer.ModelTypeTable;
+import com.google.api.codegen.transformer.ParamDocTransformer;
+import com.google.api.codegen.transformer.SurfaceNamer;
+import com.google.api.codegen.util.Name;
+import com.google.api.codegen.viewmodel.ParamDocView;
+import com.google.api.codegen.viewmodel.SimpleParamDocView;
+import com.google.api.tools.framework.model.Field;
+import com.google.api.tools.framework.model.ProtoElement;
+import com.google.api.tools.framework.model.TypeRef;
+import com.google.common.collect.ImmutableList;
+import com.google.common.collect.ImmutableSet;
+import java.util.List;
+
+public class NodeJSParamDocTransformer implements ParamDocTransformer {
```

Comment:
`NodeJSGapicContext#getApiMethodView` requires a `ParamDocTransformer`. I don't think it causes problems to have an untested implementation since it isn't used yet. When we migration NodeJS to MVVM we can fix whatever bugs are in this implementation.